mkdocs_monorepo_plugin/parser.py

```diff
@@ -41,15 +43,26 @@ def __loadAliasesAndResolvedPaths(self, nav=None):
             if type(item) is str:
                 value = str
             elif type(item) is dict:
+                key = list(item.keys())[0]
                 value = list(item.values())[0]
+                if key.startswith(WILDCARD_INCLUDE_STATEMENT):
+                    base_path, *name = key[len(WILDCARD_INCLUDE_STATEMENT):].split()
+                    key = " ".join(name)
+                    dirs = sorted(glob.glob(base_path))
+                    if dirs:
+                        value = []
+                        for dir in dirs:
+                            site = {}
+                            if os.path.exists(f"{dir}/mkdocs.yml"):
```

Comment:
Would it make more sense to do something like 
```suggestion
                            if os.path.exists(f"{dir}mkdocs.yml"):
```

and in the mkdocs.yaml something like:  `*include ./components/*/ Components` ?

That way the glob wont try files: `The ./components/randomfile/mkdocs.yml path is not valid.`?

Reply:
I originally just had it fail if you didn't have the trailing slash on `*include ./components/*/` but decided to go back in and make it work for cases where someone forgot to include it. It works either way with the existing `if os.path.exists(f"{dir}/mkdocs.yml"):` statement though. We could make a note in the user guide emphasizing the trailing slash?